languagetool-language-modules/de/src/main/resources/org/languagetool/resource/de/words_no_infix_s.txt

```diff
@@ -1418,4 +1414,183 @@ Zentrifugen
 Zionisten
 Zulagen
 Zutaten
-Zyklonen
\ No newline at end of file
+Zyklonen
+Abstrusitäten
+Achsen
+Adiabaten
+Agaven
+Ampullen
+Anekdoten
+Anleihen
+Anschriften
+Assistenten
+Astronomen
+Azteken
+Banderolen
+Basen
+Bauklötzchen
+Baumwollhemden
+Besen
+Biologen
+Chrysanthemen
+Debatten
+Debitoren
+Direktoren
+Direktorinnen
+Diätassistenten
+Drahtspeichen
+Dreiwelten
+Dänen
+Elektroden
+Enklaven
+E-Junioren
```

Comment:
Would this allow "E-Juniorenxxx" instead of "E-Junioren-Xxx"?

Reply:
Yes, but this is the same case like "T-Shirtärmel" which seems to be ok (e.g. "E-Juniorenliga")